contrib/debian/README.md

```diff
@@ -15,7 +15,7 @@ To install:
 
 If you build yourself, you will either need to modify the paths in
 the .desktop file or copy or symlink your dash-qt binary to `/usr/bin`
-and the `../../share/pixmaps/dash128.png` to `/usr/share/pixmaps`
+and the `../../share/pixmaps/bitcoin128.png` to `/usr/share/pixmaps`
```

Comment:
actually there are 2 more files referenced in /contrib/debian/dash-qt.install - bitcoin32.xpm and bitcoin16.xpm and I can't find any more references to them so renaming all 3 of them shouldn't break anything else imo